.NET/Microsoft.Recognizers.Text/Culture.cs

```diff
@@ -14,10 +15,12 @@ public sealed class Culture
         public const string Dutch = "nl-nl";
         public const string Korean = "ko-kr";
 
-        public string CultureName;
-        public string CultureCode;
+        public readonly string CultureName;
+        public readonly string CultureCode;
 
+        // The order of this array is important for GetCultureCodeWithFallback
```

Comment:
Fallback is always to English-US. Fallback should only be triggered if some culture is really not supported and the enableFallback flag is used.